apps/rotm/behaviours/back-link-check-report.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = superclass => class extends superclass {
+  getBackLink(req, res, callback) {
+    let backLink = super.getBackLink(req, res, callback);
+      if (!backLink) {
+        backLink = 'image';
```

Comment:
Question: Do the 'backlinks' get prefixed with a `/`?

Reply:
yeah it seems it is added by the getBackLink in hof somewhere

Reply:
the template?